airbyte-integrations/connectors/source-google-ads/source_google_ads/streams.py

```diff
@@ -192,3 +192,73 @@ class ShoppingPerformanceReport(IncrementalGoogleAdsStream):
     ShoppingPerformanceReport stream: https://developers.google.com/google-ads/api/fields/v8/shopping_performance_view
     Google Ads API field mapping: https://developers.google.com/google-ads/api/docs/migration/mapping#shopping_performance
     """
+
+
+class CustomQueryFullRefresh(GoogleAdsStream):
+    """
+    Class that should sync by custom user query to Google Ads API
+    Fixme: check if WHERE>start_date was applied in standard fullrefresh stream. If yes, reapply here.
+    """
+
+    def __init__(self, custom_query_config, **kwargs):
+        self.custom_query_config = custom_query_config
```

Comment:
a lot of duplication, no need to create two separate classes,
control behaviour with config